module-services/service-evtmgr/battery/BatteryController.cpp

```diff
@@ -122,18 +122,18 @@ void sevm::battery::BatteryController::poll()
 }
 void sevm::battery::BatteryController::printCurrentState()
 {
-    LOG_INFO("Charger state:%s Battery SOC %d voltage: %" PRIu32 "mV state: %s",
+    LOG_INFO("Charger state:%s Battery SOC %d voltage: %dmV state: %s",
```

Comment:
why did you removed the macro?

Reply:
Before this change, the voltage wasn't stored anywhere. I added this to `EventStore` and to be consistent now the voltage is `unsigned int` so macro is not necessary at all.

Reply:
I'd say - better have some little portability than none, this won't hurt the way it's displaying, but when (someday) anyone would like to compile it on Linux or other platform, the compile error can happen :P

Reply:
In this case, the Linux target wasn't able to compile the firmware so I had to change that. As long as the voltage is `unsigned int` it should be fine :)

Reply:
As you suggested @SP2FET and @swichu91 now the `EventStore` uses proper units from `Units.hpp` so I left `PRIu32` as it was before. I pushed an additional commit.